mpc/walk_ocp.py

```diff
@@ -80,6 +53,30 @@
         + [[1, 1]] * 40
         + [[1, 1]]
     )
+# #####################################################################################
+# ### LOAD ROBOT ######################################################################
+# #####################################################################################
+
+# ## LOAD AND DISPLAY TALOS
+# Load the robot model from example robot data and display it if possible in
+# Gepetto-viewer
+
+urdf = robexLoadAndReduce("talos", walkParams.robotName)
+robot = RobotWrapper(urdf.model, contactKey="sole_link")
```

Comment:
this `robot` was defined on the old line 32, and is now defined on the new line 65.
But it is used at old 63 / new 36. Could you fix this order ?